cubedash/templates/dataset.html

```diff
@@ -139,9 +144,15 @@ <h3>
                                     <i class="fa fa-caret-right" aria-hidden="true"></i>
                                 </th>
                                 <td>
-                                    <a href="{{ offset | resolve_remote_url(uri) }}"
+                                    {% set offset_url = offset | resolve_remote_url(uri) %}
+                                    <a href="{{ offset_url }}"
                                        class="uri-path">
                                         {{ offset }}
+                                        {% if 'thumbnail:nbart' == name or 'thumbnail' == name %}
+                                            {# workaround for jinja preventing global varaibles from being updated inside
+                                                for loops and if/else statements #}
+                                            {% set _ = thumbnail_url.append(offset_url) %}
+                                        {% endif %}
```

Comment:
Eeep, workarounds :(

See below for a nicer option, but open to your thoughts to.